buildozer/def.bzl

```diff
@@ -77,13 +77,12 @@ _buildozer = rule(
             default = True,
         ),
         "_buildozer": attr.label(
-            default = Label("@com_github_bazelbuild_buildtools//buildozer"),
+            default = "@com_github_bazelbuild_buildtools//buildozer",
             cfg = "host",
-            allow_single_file = True,
             executable = True,
```

Comment:
`executable = True` implies and also actually allows specifying files so `allow_single_file = True` is not needed.